src/test/java/com/microsoft/spring/data/gremlin/repository/ServiceRepositoryIT.java

```diff
@@ -184,5 +184,31 @@ public void testServiceFindById() {
         Assert.assertEquals(foundConfig.get(), this.config);
         Assert.assertEquals(foundEureka.get(), this.eureka);
     }
+
+    @Test
+    public void testServiceFindByNameAndInstanceCount() {
+        this.repository.save(config);
+        this.repository.save(eureka);
```

Comment:
Wonder is it a problem or not, it's about naming and test data. The `config` and `eureka` which may not be straightforward enough for a `gremlin ONLY` user.

Reply:
how about `configDomain` and `eurekaDomain` ?